test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) Registered app
+-- 4) Activated app
```

Comment:
It would be better to reword in a way `App is registered`, `App is activated` etc.